components/header.js

```diff
@@ -0,0 +1,26 @@
+import styled from "styled-components";
+
+export default function Header() {
+  return (
+    <StyledHeader>
+      <Title>CommuniFI</Title>
+    </StyledHeader>
+  );
+}
+
+const StyledHeader = styled.div`
+  background-color: #cfd6de;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 20px;
+  position: fixed;
+  top: 0px;
+  left: 0px;
+  right: 0px;
+  padding: 0px 20px 0px 20px;
```

Comment:
You could write this shorter
```suggestion
  padding: 0px 20px;
```

If you have the same values for top/bottom and right/left padding.